server/src/internalClusterTest/java/org/elasticsearch/discovery/ClusterDisruptionIT.java

```diff
@@ -94,18 +93,11 @@ static ConflictMode randomMode() {
     }
 
     /**
-     * Test that we do not loose document whose indexing request was successful, under a randomly selected disruption scheme
+     * Test that we do not lose documents, indexed via requests that return success, under randomly selected disruption schemes.
      * We also collect &amp; report the type of indexing failures that occur.
      * <p>
-     * This test is a superset of tests run in the Jepsen test suite, with the exception of versioned updates
+     * This test is a superset of tests run in the Jepsen test suite, with the exception of versioned updates.
      */
-    @TestIssueLogging(
```

Comment:
I would suggest leaving this logging in place, although recording it as `@TestIssueLogging` for #41068 is clearly not accurate. Let's make it a `@TestLogging` with `reason = "if this test fails we need all this logging to understand the failure"` or similar.